src/platform_impl/android/mod.rs

```diff
@@ -475,7 +475,9 @@ impl<T: 'static> EventLoop<T> {
                                 phase,
                                 location,
                                 id: pointer.pointer_id() as u64,
-                                force: None,
+                                force: Some(crate::event::Force::Normalized(
+                                    pointer.pressure() as f64
+                                )),
```

Comment:
Could you import it at the top properly?

Reply:
Updated